src/components/containers/ProjectCard.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+
+import {
+    CardContent,
+    Typography,
+    CardMedia,
+    Card,
+    makeStyles,
+    Box,
+    Grid
+}
```

Comment:
It looks like this file hasn't been linted

Reply:
I made sure to lint it